web/src/components/EventsDateFilter.tsx

```diff
@@ -66,17 +29,24 @@ const EventsDateFilter = ({
   setEndDate,
 }: EventsDateFilterProps): ReactElement => {
   const [showDateFilter, setShowDateFilter] = useState(false)
+  const { mobile } = useDimensions()
   const { t } = useTranslation('events')
 
   const today = DateTime.now()
   const inAWeek = DateTime.now().plus({ week: 1 })
+  const formattedStartDate = startDate?.toFormat('dd.MM.yyyy') ?? '∞'
+  const formattedEndDate = endDate?.toFormat('dd.MM.yyyy') ?? '∞'
 
   return (
     <>
-      <StyledAccordion disableGutters expanded={showDateFilter} elevation={0}>
-        <FilterToggle isDateFilterActive={showDateFilter} setToggleDateFilter={setShowDateFilter} />
+      <Accordion disableGutters expanded={showDateFilter} elevation={0}>
```

Comment:
by default Accordion uses `h3` element for heading, it should be overriden with `slotProps` to use another component

Reply:
What should be used instead?

Reply:
I was not sure about it in our case, but I do not have any other opinion on it :+1: